crates/bevy_ecs/src/schedule/config.rs

```diff
@@ -267,6 +269,10 @@ where
     fn ambiguous_with_all(self) -> Config {
         self.into_config().ambiguous_with_all()
     }
+    /// Always run this system when the schedule is running in system step mode.
+    fn ignore_stepping(self) -> Config {
```

Comment:
I think that this should be something end users can mutate freely? One-way only is rough for a debugging tool.

Reply:
Agreed